tests/kernel/blktests.pm

```diff
@@ -17,6 +17,26 @@ use testapi;
 use utils;
 use repo_tools 'add_qa_head_repo';
 
+#can accept tags: none, all, specific
+sub _test_devices {
```

Comment:
I think `configure_tests_devices` would be better name for this function.

Reply:
or list_tests_devices or enumerate_tests_devices; I was having originally this sub named configure_tests_devices but changed it as it does not configure, just lists devices to be used for testing.

Reply:
I settled for prepare_blktests_config. I will mark this resolved for ease of review